src/OrchardCore/OrchardCore.FileStorage.AmazonS3/AmazonWebServiceResponseExtensions.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Amazon.Runtime;
+using System.Net;
+
+namespace OrchardCore.FileStorage.AmazonS3;
+
+internal static class AmazonWebServiceResponseExtensions
+{
+    public static bool IsSuccessful(this AmazonWebServiceResponse response) =>
+        response.HttpStatusCode == HttpStatusCode.OK;
+
+    public static bool DeletedSuccessfully(this AmazonWebServiceResponse response) =>
```

Comment:
Just to be in line with `IsSuccessful`:

```suggestion
    public static bool IsDeleteSuccessful(this AmazonWebServiceResponse response) =>
```